packages/components/pie-notification/test/component/pie-notification.spec.ts

```diff
@@ -17,4 +39,222 @@ test.describe('PieNotification - Component tests', () => {
         // Assert
         expect(notification).toBeVisible();
     });
+
+    test.describe('Props', () => {
+        test.describe('variants', () => {
+            variants.forEach((variant) => {
+                test(`should render when the variant is ${variant}`, async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieNotification, {
+                        props: {
+                            variant,
+                        },
+                    });
+
+                    // Act
+                    const notification = page.locator(componentSelector);
```

Comment:
Out of interest, are these equivalent?
```suggestion
                    // Arrange & Act
                    const notification = await mount(PieNotification, {
                        props: {
                            variant,
                        },
                    });
```

Reply:
I am not sure, my tests fail when I apply this suggestion.